Sources/Core/Extensions/Other/XCTestCase+PovioKit.swift

```diff
@@ -12,9 +12,38 @@ public extension XCTestCase {
   func XCTAssertEqualURLRequest(_ lhs: URLRequest?, _ rhs: URLRequest?, file: StaticString = #filePath, line: UInt = #line) {
     XCTAssertEqual(lhs, rhs, file: file, line: line)
     guard let lhs = lhs, let rhs = rhs else { return }
-    XCTAssertEqual(lhs.url, rhs.url, "url", file: file, line: line)
-    XCTAssertEqual(lhs.httpMethod, rhs.httpMethod, "httpMethod", file: file, line: line)
-    XCTAssertEqual(lhs.httpBody ?? Data(), rhs.httpBody ?? Data(), "httpBody", file: file, line: line)
-    XCTAssertEqual(lhs.allHTTPHeaderFields ?? [:], rhs.allHTTPHeaderFields ?? [:], "allHTTPHeaderFields", file: file, line: line)
+    XCTAssertEqual(lhs.url, rhs.url, "Url doesn't equal!", file: file, line: line)
+    XCTAssertEqual(lhs.httpMethod, rhs.httpMethod, "HttpMethod doesn't equal!", file: file, line: line)
+    XCTAssertEqual(lhs.httpBody ?? Data(), rhs.httpBody ?? Data(), "HttpBody doesn't equal!", file: file, line: line)
+    XCTAssertEqual(lhs.allHTTPHeaderFields ?? [:], rhs.allHTTPHeaderFields ?? [:], "AllHTTPHeaderFields doesn't equal!", file: file, line: line)
+  }
+  
+  func XCTAssertEqualImage(_ lhs: UIImage?, _ rhs: UIImage?, file: StaticString = #filePath, line: UInt = #line) {
+    guard let image1 = lhs, let image2 = rhs,
+          let imageData1 = image1.pngData(),
+          let imageData2 = image2.pngData() else { return }
+    XCTAssertTrue(imageData1.elementsEqual(imageData2), "Images doesn't equal!", file: file, line: line)
+  }
+  
+  func XCTAssertNotEqualImage(_ lhs: UIImage?, _ rhs: UIImage?, file: StaticString = #filePath, line: UInt = #line) {
+    guard let image1 = lhs, let image2 = rhs,
+          let imageData1 = image1.pngData(),
+          let imageData2 = image2.pngData() else { return }
+    XCTAssertFalse(imageData1.elementsEqual(imageData2), "Images do equal!", file: file, line: line)
+  }
+  
+  func XCTAssertEqualFont(lhs: UIFont?, rhs: UIFont?, file: StaticString = #filePath, line: UInt = #line) {
```

Comment:
Let's omit the argument labels like in other methods.

Reply:
We actually need them, otherwise error is not propagated forward.

Reply:
I am talking about `lhs: UIFont?, rhs: UIFont?` -> `_ lhs: UIFont?, _ rhs: UIFont?` not `file: StaticString = #filePath, line: UInt = #line` , or am I missing something?

Reply:
Missed that one, thanks 👌